docs/0.1.0/install_macos.md

```diff
@@ -2,128 +2,184 @@
 
 ## Prerequisites
 
-1. Access to a synched RSK node. You can do this in a variety of ways:
-	1. Run your own node on TestNet or MainNet, see [https://github.com/rsksmart/rskj/wiki/Install-RskJ-and-join-the-RSK-Orchid-Mainnet-Beta](https://github.com/rsksmart/rskj/wiki/Install-RskJ-and-join-the-RSK-Orchid-Mainnet-Beta)
-	2. Compile and run a RSK node locally, see [https://github.com/rsksmart/rskj/wiki/Compile-and-run-a-RSK-node-locally](https://github.com/rsksmart/rskj/wiki/Compile-and-run-a-RSK-node-locally)
-2. An RSK account with an RBTC balance NOT lower than 0.001 RBTC
-3. XCode
+1. Access to a synced RSK node. You can do this in a variety of ways:
+	1. Run your own node on Testnet or Mainnet, see [Node (RSKj): Install](https://developers.rsk.co/rsk/node/install/).
+	2. Compile and run a RSK node locally, see [Node (RSKj): Contribute](https://developers.rsk.co/rsk/node/contribute/).
```

Comment:
i'd like to add here—as we have in the Ubuntu guide—which is the oldest version of MacOS that will support Lumino and this installation method. unfortunately, i do not know which version this is.

this can be tackled through [RLP-1018](https://jirainfuy.atlassian.net/browse/RLP-1018).

it's a low priority issue.

Reply:
i agree, but don't worry to much about the oldest version of MacOS since Apple doesn't support too old MacOS versions, they always have all the macs using the latest so i think we are good with that.